File: app/src/main/java/org/mifos/selfserviceapp/models/ChargeCalculationType.java
package org.mifos.selfserviceapp.models;

/**
 * Created by michaelsosnick on 12/11/16.
 */

public class ChargeCalculationType {
    private int id;
    private String code; // example "chargeCalculationType.flat"
    private String value;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}

File: app/src/main/java/org/mifos/mobile/models/ChargeCalculationType.kt
package org.mifos.mobile.models

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * Created by michaelsosnick on 12/11/16.
 */

@Parcelize
data class ChargeCalculationType(
        var id: Int = 0,
        var code: String? = null, // example "chargeCalculationType.flat"
        var value: String? = null
) : Parcelable